content/embeds/tryout-redisai.md

```diff
@@ -0,0 +1,199 @@
+## Getting Started
```

Comment:
@bmansheim do we add this to others, the docker run?  I don't think we should do this here.  The idea is to get them to do it on redis enterprise

Reply:
@K-Jo We didn't do it for other modules but currently we don't have installation instructions so maybe we change it when we add the installation instructions.

Reply:
No we leave it out. thanks